RELEASES.md

```diff
@@ -1,4 +1,26 @@
 # [Unreleased]
+- Backward-incompatible piece of [#1233](https://github.com/paritytech/ink/pull/1233): removal of `eth_compatibility crate`
+- Backward-incompatible piece of [#1224](https://github.com/paritytech/ink/pull/1224): dependency on `[seal1] seal_set_storage`
+- Backward-incompatible "Optimise deny_payment. Use eerywhere semantic of deny. ([#1267](https://github.com/paritytech/ink/pull/1267))"
+  (see @HCastano SE [answer](https://substrate.stackexchange.com/a/3000/472) in this regard)
+
+# Version 3.3.0
+
+This release is to make *ink!* 3.x.x backward compatible with *substrate-contracts-node* version [0.13.0](https://github.com/paritytech/substrate-contracts-node/releases/tag/v0.13.0) again. 
```

Comment:
This is just one of the reasons, we should also mention that we didn't follow SemVer

Reply:
We're trying to alleviate two things here, incompatibility between 3.2 and earlier 3.x releases, as well as incompatibility with the 0.13 node